docs/src/SUMMARY.md

```diff
@@ -29,6 +29,7 @@ Summary
   - [Settings](guides/settings.md)
   - [Teleburning](guides/teleburning.md)
   - [Testing](guides/testing.md)
+  - [API](guides/api.md)
```

Comment:
Can we put this as the first entry to the guides section?

Reply:
sure thing